src/lib/CMakeLists.txt

```diff
@@ -16,4 +16,3 @@ ADD_SUBDIRECTORY(ui)
 ADD_SUBDIRECTORY(qt)
 ADD_SUBDIRECTORY(graphics)
 ADD_SUBDIRECTORY(network)
-ADD_SUBDIRECTORY(oiio)
```

Comment:
@bernie-laberge This change was most likely done at the beginning of my work. I'm a bit suspicious: if we don't include this folder: should we remove it? (like IOJP2) I'm trying to think and it might have been wrong to remove it. I think I removed it because we don't use src/pub/OpenImageIO anymore. Then again, nothing seems to fail thus maybe the code in there is now in the OIIO lib we download during the build?